css/joomla.css

```diff
@@ -375,3 +375,13 @@ ul#civicrm-menu li#crm-qsearch {
 .crm-container .disabled {
   font-weight: normal;
 }
+
+#crm-container .crm-container .form-layout td.label {
+  width: inherit;
+}
+
+#crm-container .label {
```

Comment:
Using the class selector `.crm-container` will include popups. The id selector `#crm-container` is only for the main content area.